src/main/features/eligibility/views/macro/mcolSupport.njk

```diff
@@ -0,0 +1,9 @@
+{% macro mcolSupportWidget(label = 'Help and advice') %}
+  <h2 class="heading-medium">{{ t(label) }}</h2>
+  <pre>{{ t('Money Claim Online helpdesk') }}</pre>
+  <pre>{{ t('Email: ccbc@hmcts.gsi.gov.uk') }}</pre>
+  <pre>{{ t('Telephone: 0300 123 1057') }}</pre>
+  <pre>{{ t('Monday to Friday, 8:30am to 5pm') }}</pre>
+  <p><a target="_blank" href="https://www.gov.uk/call-charges">{{ t('Find out about call charges') }}
```

Comment:
Please use `views/macro/externalLink.njk` macro for external link.

Reply:
I have created a macro in common-frontend because supportWidget already exists there. externalLink is in citizen-frontend, so don't want to import it to common-frontend.